front/src/core-logic/domain/auth/auth.slice.ts

```diff
@@ -9,23 +9,34 @@ export type FederatedIdentityWithUser = FederatedIdentity & {
 
 interface AuthState {
   federatedIdentityWithUser: FederatedIdentityWithUser | null;
+  isPeConnected: boolean;
 }
 
 const initialState: AuthState = {
   federatedIdentityWithUser: null,
+  isPeConnected: false,
 };
 
 const onFederatedIdentityReceived = (
   state: AuthState,
   action: PayloadAction<FederatedIdentityWithUser | null>,
 ) => {
   state.federatedIdentityWithUser = action.payload;
+  if (
+    state.federatedIdentityWithUser?.provider === "peConnect" &&
+    state.federatedIdentityWithUser.token !== authFailed
+  ) {
+    state.isPeConnected = true;
+  }
 };
 
 export const authSlice = createSlice({
   name: "auth",
   initialState,
   reducers: {
+    setIsPeConnected: (state, action: PayloadAction<boolean>) => {
```

Comment:
```suggestion
    changePeConnectedStatusRequested: (state, action: PayloadAction<boolean>) => {
```

Ou quelque chose du genre